<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtlAddRoomPage</class>
 <widget class="QDialog" name="HtlAddRoomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_top">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add New Room</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="IdLabel">
      <property name="text">
       <string>Id</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="TypeLabel">
      <property name="text">
       <string>Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="DescriptionLabel">
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="PriceLabel">
      <property name="text">
       <string>Original Price</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="DiscountLabel">
      <property name="text">
       <string>Discount</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="IdLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="TypeLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="DescriptionLineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="PriceLineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="DiscountLlineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ReadFileBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>from &quot;rooms.txt&quot;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
